rero_ils/modules/monitoring/views.py

```
@@ -230,14 +229,17 @@ def missing_pids(doc_type):
-    :return: jsonified redis info.
+    :return: jsonified redis info keyed by instance name.
     """
-    url = current_app.config.get("ACCOUNTS_SESSION_REDIS_URL", "redis://localhost:6379")
-    redis = Redis.from_url(url)
-    info = redis.info()
-    return jsonify({"data": info})
+    data = {}
+    for name, client in current_app.extensions.get("rero_ils_redis_instances", {}).items():
+        try:
+            data[name] = {"name": name, **client.info()}
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)